### DISCUSSION TITLE
referensi untuk menghandle tulisan berbeda yang berdempetan

### DISCUSSION THREAD

**Discussion by Gustana** [teamalgoritma/community#698] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/23212456/d0277cef-770e-458e-af44-d795d6c291bf)

dalam konteks nlp, bagaimana cara untuk memberbaiki tulisan yang berdempetan seperti pada gambar?

**Comment by nadyadtm** [comment]
Dear Mas @Gustana 

Untuk toolsnya apakah lebih prefer R atau python?

**Reply by Gustana** [reply]
python kak

**Reply by nadyadtm** [reply, marked as answer]
Baik mas, untuk memperbaiki tulisan tersebut, ada sebuah teknik yang bernama Word Segmentation. Jika ingin mengeksplorasi toolsnya di python dapat menggunakan package yang bernama `wordsegmentation`. Untuk mencobanya dapat mengakses link berikut https://pypi.org/project/wordsegmentation/. Namun untuk saat ini hanya tersedia pada teks berbahasa inggris saja.

Semoga membantu
